content/_index.md

```diff
@@ -39,6 +39,6 @@ features:
     feature_matrix:
       data: gadgets
       title: The Gadgets
-      learn_more_link: https://www.inspektor-gadget.io/docs/latest/gadgets/
+      learn_more_link: https://www.inspektor-gadget.io/docs/latest
```

Comment:
Better to point to https://www.inspektor-gadget.io/docs/latest/builtin-gadgets/?

Reply:
Once we remove the built-in gadgets we'll have to change a lot of the website. I'll approve it anyway.